Source/InMobiAdapter.swift

```diff
@@ -67,7 +67,10 @@ final class InMobiAdapter: NSObject, PartnerAdapter {
     /// - parameter completion: Closure to be performed with the fetched info.
     func fetchBidderInformation(request: PreBidRequest, completion: @escaping ([String : String]?) -> Void) {
         log(.fetchBidderInfoStarted(request))
-        guard let bidToken = IMSdk.getToken() else {
+        let extras = ["tp": "c_chartboost", "tp-ver": Helium.sdkVersion]
+        // The `andKeywords` parameter is type `String?` so we're not passing it `request.keywords`.
+        // TBD whether it would be helpful to flatten our `[String: String]` keywords and send that.
```

Comment:
I think it's safe to assume we won't do this. We didn't do it for Google.
If you want to be super safe you can create a ticket to wait until this question is solved, but we can remove this comment here I think.